apps/src/templates/sectionAssessments/sectionAssessmentsRedux.js

```diff
@@ -637,7 +637,7 @@ export const countSubmissionsForCurrentAssessment = (state) => {
     const studentResponses = getAssessmentResponsesForCurrentScript(state);
     let totalSubmissions = 0;
     Object.values(studentResponses).forEach((student) => {
-      if (Object.keys(student.responses_by_assessment).includes(currentAssessmentId.toString())) {
+      if (Object.keys(student.responses_by_assessment).includes(currentAssessmentId + '')) {
```

Comment:
This may be the safest change, but it seems reasonable to step back a layer and make `countSubmissionsForCurrentAssessment` early-out and return 0 if `state.sectionAssessments.assessmentId` is undefined.  It'd be an easy case to add a unit test for, too.